packages/introspect/src/e2e.test.ts

```diff
@@ -1,6 +1,24 @@
 import { describe, expect, test } from 'vitest';
-import { generate } from './generate';
 import * as prettier from 'prettier';
+import { Pool, types } from 'pg';
+import { generate } from './generate';
+import {
+    createTableQuery,
+    deleteTableQuery,
+    insertTableQuery,
+    selectTableQuery,
+} from './tests/sqlQueries';
+
+// Use the OIDs imported from pg.types to set custom type parsers
+types.setTypeParser(types.builtins.DATE, (value) => value);
```

Comment:
Why are you putting this here?

Reply:
To replicate the same behaviour of parsing to string that we have in the PgExecutor... that will allow for the schema matching to be correct